lib/shipitron/server/docker/run_build_script.rb

```diff
@@ -10,15 +10,14 @@ class RunBuildScript
         required :application
         required :docker_image
         required :git_sha
-        optional :build_script
-
-        before do
-          context.build_script ||= 'shipitron/build.sh'
-        end
+        required :named_tag
+        optional :build_script, default: 'shipitron/build.sh'
+        optional :registry
 
         def call
           Logger.info 'Building docker image'
 
+          docker_image.registry = registry if registry != nil
```

Comment:
What would registry be if it isn't set?

Reply:
Nothing! The docker tools all default to docker hub if you don't set anything.